doc/contribute/contributor_expectations.rst

```diff
@@ -213,17 +213,27 @@ defines the following escalation process for resolving technical disagreements.
 
   - Assignee to act as moderator if applicable.
 
-- Optionally resolve in the next `Zephyr Dev Meeting`_ or `Architecture Working
-  Group`_ meeting with more Maintainers and project stakeholders.
+- Resolve in the next `Zephyr Dev Meeting`_  meeting with more
```

Comment:
Please leave the "optionally" (and ideally the whole paragraph as it was). As you @nashif said not everybody can join these meetings. We don't all live in US time. It is hardly helpful to demand that 2 people in Europe, India or Australia need to join a meeting in US pacific time in which things may or may not be discussed, in which there is high likelihood nobody else will care about the issue.
Having the dev-review and architecture meeting as an option/possibility for discussion is good. Forcing it as the path, no.